--- Configurations/10-main.conf.orig	2019-09-10 09:13:07.000000000 -0400
+++ Configurations/10-main.conf	2020-01-27 13:16:41.992273600 -0500
@@ -1252,7 +1252,7 @@
                                        })),
         defines          => add(picker(default => [], # works as type cast
                                        debug   => [ "DEBUG", "_DEBUG" ])),
-        lib_cflags       => add(sub { $disabled{shared} ? "/MT /Zl" : () }),
+        lib_cflags       => add(sub { $disabled{shared} ? "/MD" : () }),
         # Following might/should appears controversial, i.e. defining
         # /MDd without evaluating $disabled{shared}. It works in
         # non-shared build because static library is compiled with /Zl
@@ -1265,7 +1265,7 @@
         # prefer [non-debug] openssl.exe to be free from Micorosoft RTL
         # redistributable.
         bin_cflags       => add(picker(debug   => "/MDd",
-                                       release => sub { $disabled{shared} ? "/MT" : () },
+                                       release => sub { $disabled{shared} ? "/MD" : () },
                                       )),
         bin_lflags       => add("/subsystem:console /opt:ref"),
         ex_libs          => add(sub {
@@ -1335,7 +1335,7 @@
             combine('/GF /Gy',
                     sub { vc_wince_info()->{cflags}; },
                     sub { `cl 2>&1` =~ /Version ([0-9]+)\./ && $1>=14
-                              ? ($disabled{shared} ? " /MT" : " /MD")
+                              ? ($disabled{shared} ? " /MD" : " /MD")
                               : " /MC"; }),
         cppflags         => sub { vc_wince_info()->{cppflags}; },
         lib_defines      => add("NO_CHMOD", "OPENSSL_SMALL_FOOTPRINT"),
